napalm/ios/ios.py

```diff
@@ -1955,6 +1955,54 @@ def get_snmp_information(self):
             snmp_dict['chassis_id'] = snmp_chassis
         return snmp_dict
 
+    def get_users(self):
+        """
+        Returns a dictionary with the configured users.
+        The keys of the main dictionary represents the username.
+        The values represent the details of the user,
+        represented by the following keys:
+
+            * level (int)
+            * password (str)
+            * sshkeys (list)
+
+        *Note: sshkeys on ios is the ssh key fingerprint
+
+        The level is an integer between 0 and 15, where 0 is the
+        lowest access and 15 represents full access to the device.
+        """
+        users = {}
+        command = "show run | section username"
+        output = self._send_command(command)
+        curuser = ""
+        for line in output.splitlines():
+            if line.startswith("username"):
+                tokens = line.split(" ")
+                users[tokens[1]] = {
+                    'level': 1,
+                    'password': "",
+                    'sshkeys': []
+                }
+                i = 2
+                while i < len(tokens):
+                    if tokens[i] == "privilege":
+                        users[tokens[1]]["level"] = int(tokens[i+1])
+                        i += 2
+                    elif tokens[i] == "secret":
+                        users[tokens[1]]["password"] = tokens[i+2]
+                        i += 3
+            elif line.startswith(" "):
```

Comment:
I think we should do regex here as well and have a newline in the regex pattern (once again I can help with this).